buildsystem/versions.gradle

```diff
@@ -44,7 +44,7 @@ ext {
             play_services_auth              : '17.0.0',
             retrofit                        : '2.6.2',
             status_keycard                  : '3.0.1',
-            svalinn                         : 'v0.15.0',
+            svalinn                         : 'aa5b18af90',//'v0.15.0',
```

Comment:
dangling left

Reply:
svalinn changes are ongoing. There will be a pr also